services/gateway/apis/http-version3/swagger/parameters/tokens.json

```
@@ -0,0 +1,11 @@
+{
+    "tokenID": {
+        "name": "tokenID",
+        "in": "query",
+        "description": "Token ID to query",
+        "type": "string",
+        "format": "id",
+        "minLength": 1,
+        "maxLength": 8
```

Please also make the same change in the gateway API def.
```suggestion
        "minLength": 8,
        "maxLength": 8
```